configs/dev/portal-ext.properties

```diff
@@ -1,4 +1,6 @@
 include-and-override=portal-developer.properties
+users.screen.name.special.characters=@_.
```

Comment:
You're duplicating this config in all folders.
Instead of doing it like this, you can add it to the `configs/common` folder.
Files added to `configs/common` are applied to all enviroments.

Reply:
Duplicated config files delete from all environment folders and added to configs/common